src/app/pages/charging-stations/charging-limit/charging-profile-limit/charging-station-charging-profile-limit-chart.component.ts

```diff
@@ -27,10 +36,16 @@ export class ChargingStationSmartChargingLimitPlannerChartComponent implements O
   @Input() public connectorId!: number;
   @Input() public chargingSchedules: Schedule[];
 
-  @ViewChild('primary', {static: true}) public primaryElement!: ElementRef;
+  @ViewChild('primary', { static: true }) public primaryElement!: ElementRef;
   @ViewChild('danger', { static: true }) public dangerElement!: ElementRef;
-  @ViewChild('chart', {static: true}) public chartElement!: ElementRef;
+  @ViewChild('chart', { static: true }) public chartElement!: ElementRef;
+
+  public selectedUnit = ConsumptionUnit.KILOWATT;
 
+  public unitMap = [
```

Comment:
It's used in more than one file, it should be in /shared

Reply:
How do you mean this. Where and how would you store the Array?